src/components/rangeUsePlanPage/pageForAH/index.js

```diff
@@ -374,7 +374,7 @@ class PageForAH extends Component {
           {canEdit ? (
             <EditableMinisterIssues {...ministerIssueProps} />
           ) : (
-            <MinisterIssues {...ministerIssueProps} />
+            <MinisterIssues issues={plan.ministerIssues} /> //  TODO: these should be populated objects instead of ids
```

Comment:
does this TODO need to be addressed for things to work or is it a 'we should clean this up' kind of TODO?